akka-cluster-tools/src/main/resources/reference.conf

```diff
@@ -138,6 +138,9 @@ akka.cluster.singleton {
 }
 
 akka.cluster.singleton-proxy {
+  # The actor name of the singleton actor that is started by the ClusterSingletonManager
+  singleton-name = ${akka.cluster.singleton.singleton-name}
```

Comment:
This feels somehow a little off (but might be okay, depends on the explanation): The reason was to remove duplication so that the configuration is more “in one place” than before, therefore I kind-of expected this configuration to be taken directly from the `akka.cluster.singleton` section, since now the user need to override two config keys with identical contents (`reference.conf` is resolved fully before being used as fallback, so this alias does not expand to users’ overridden values).